proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+│ a82585e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ c778a78        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 551082e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 15bef96        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+└────────────────┴──────────────┴─────────┴───────────────┴──────────────┴────────────────────────┴──────────────────┴────────────┴────────────────────┴────────────────────┴───┘
+```
+
+This table includes parameters and metrics from the `evaluate` stage and all
+stages upstream of it.  It's possible to also have a `--single-item` argument to
+exclude upstream changes, but this is out of scope for now.
+
+# How We Teach This
+
+See [Detailed design](#detailed-design) for a proposed help message.
+
+This is consistent with other commands, and it's only one additional argument to
+an existing command, so it likely doesn't require much teaching. It might be
+necessary to explicitly add that parameters and metrics from upstream stages
+will be shown.
+
+# Drawbacks
+
+This proposal may require changes to how DVC collects and tracks parameters and
+metrics. Implementation could break existing functionality, and it is
+inconsistent with the current behaviors of `dvc params`, `dvc metrics`, and `dvc
+exp diff` commands.
+
+Further, this proposal adds to an already crowded set of `dvc exp show` options,
+and there is no single way to show parameters and metrics that's going to work
+for every use case. For example, this proposal does nothing to filter out
+parameters for upstream stages, which may frequently be irrelevant if only the
+stages near the end of the pipeline change during experimentation.
+
+# Alternatives
+
+There are many alternative methods to refine the output of `dvc exp show`:
+
+* Current functionality: `exp show` already has options to include/exclude any
+  column, so the flexibility to get the same output as suggested here already
+  exists.
+* Expand the include/exclude options to accept wildcard/glob/regex arguments for
+  more flexibility to exclude an entire file or section of a file like
+  `exclude-params=params.json:local_config*`.
+* Save configuration options for `dvc exp show` so they can be reused and
+  modified without typing out all options at the command line each time.
+* Add a tsv/csv output format option in `dvc exp show` to enable users to
+  manipulate the table with other tools.
```

Comment:
➕

Reply:
:heavy_plus_sign:

Reply:
Although... Since there's already `--show-json`, TSC/CSV may be too much (you can easily [convert](https://json-csv.com/) from JSON to tabular).

Reply:
> Although... Since there's already `--show-json`, TSC/CSV may be too much (you can easily [convert](https://json-csv.com/) from JSON to tabular).

This isn't true for `dvc exp show`, since the json is not structured like tabular data. Try it out and the results are kind of a mess (too messy to post in a comment here). The json could be cleaned up to make this easy, but it was specifically designed as output for downstream apps to consume instead of as a format that users can easily understand. So I guess it's less about having a csv format specifically, and more about having an easily understood output format that can be saved and parsed.